Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+  agent { dockerfile { } }
+  options {
+    ansiColor('xterm')
+  }
+  stages {
+    stage("Init title") {
+      when { changeRequest() }
+      steps {
+        script {
+          currentBuild.displayName = "PR ${env.CHANGE_ID}: ${env.CHANGE_TITLE}"
+        }
+      }
+    }
+    stage('Initialize Git/SSH') {
+      steps {
+        sshagent(['2b3d8d6b-0855-4b59-864a-6b3ddf9c9d1a']) {
+          sh '''
+            git config --global user.email "admin@runtimeverification.com"
+            git config --global user.name  "RV Jenkins"
+            mkdir -p ~/.ssh
+            cp tools/ssh_config ~/.ssh/config
+            chmod go-rwx -R ~/.ssh
+            ssh github.com || true
+          '''
+        }
+      }
```

Comment:
This should happen in the Dockerfile instead. See https://github.com/runtimeverification/firefly/blob/master/Dockerfile#L58